### ISSUE TITLE
Merge NGINX Plus Ansible role into NGINX OSS Ansible role?

### ISSUE THREAD

**Issue by alessfg** [nginxinc/ansible-role-nginx-oss#5]
Should we merge the Plus role into the OSS role and let users specify which version of NGINX they want to deploy via a variable or should we keep both roles separate? Personally I favor the former but I want to open up this discussion to the community.

**Comment by inka** [comment]
pro merge. As it simplifies handling in hybrid environments there you have to maintain OSS and PLUS in parallel.

**Comment by gooodmorningopenstack** [comment]
I would invoke the DRY concept because both OSS and Plus version will share common patterns. It complicates a little the role but @inka got a good point also for hybrid environment.

Last point is, there are plenty of good Nginx roles out there, we need to do more or different. It can be one of it.

**Comment by alessfg** [comment]
The open source NGINX and NGINX Plus roles have been merged into a new [NGINX Ansible role](https://github.com/nginxinc/ansible-role-nginx).
